deltachat-ios/Controller/ChatListViewController.swift

```diff
@@ -746,11 +746,17 @@ class ChatListViewController: UITableViewController {
         if selected == 0 {
             return
         }
-        let alert = UIAlertController(
-            title: nil,
-            message: String.localized(stringID: "ask_delete_chat", parameter: selected),
-            preferredStyle: .safeActionSheet
-        )
+
+        let message: String
+        if selected == 1,
```

Comment:
Maybe it'd make sense to put these magic numbers into more verbose constants?

Reply:
it is not a magic number, it is just a counter. but the variable name could be more verbose

Reply:
Ah! Now it makes sense 🙈

Reply:
i changed the variable name from `selected` to `selectedCount`, that should make things clearer